aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -789,6 +791,30 @@ Tensor min(const Tensor& self, const Tensor& other) {
 
 Tensor& min_(Tensor& self, const Tensor& other) { return at::min_out(self, self, other); }
 
+Tensor& maximum_out(Tensor& result, const Tensor& self, const Tensor& other) {
+  auto iter = TensorIterator::binary_op(result, self, other,
+                                        /*check_mem_overlap=*/true);
+  maximum_stub(iter.device_type(), iter);
+  return result;
+}
+
+Tensor maximum(const Tensor& self, const Tensor& other) {
+  Tensor result = at::empty(0, self.options());
```

Comment:
`Tensor result;`

Reply:
Thanks for pointing this out, I have changed the code to `Tensor result` and used `iter.output()` for the returning value.